python/tests/conftest.py

```diff
@@ -203,6 +204,15 @@ def create_version_store(
 
     try:
         yield create_version_store
+    except RuntimeError as e:
+        if "mdb_" in str(e): # Dump keys
```

Comment:
What's this?

Reply:
`mdb_` is the prefix that LMDB would have in its error messages. There's no specific exception I can catch here.